draft-ietf-rtcweb-jsep.xml

```diff
@@ -454,6 +454,27 @@ setRemote(ANSWER) |                                   |
                 application will not need to read or change them.</t>
 
             </section>
+            <section title="Session Description Versions"
+                     anchor="sec.session-description-vers">
+              <t>
+                The SessionDescription contains a "session-version"
+                field which is required by <xref target="RFC4566"/> to increase with
```

Comment:
I think 4566 only requires it to change not increment but I think we should say it needs to increment because that makes things easier.